tests/src/test/scala/com/nvidia/spark/rapids/FuzzerUtils.scala

```diff
@@ -355,22 +338,26 @@ class EnhancedRandom(protected val r: Random, protected val options: FuzzerOptio
     new Timestamp((futureDate * r.nextDouble()).toLong)
   }
 
-  private def generateString(): String = {
-    if (options.asciiStringsOnly) {
-      val b = new StringBuilder()
-      for (_ <- 0 until options.maxStringLen) {
-        b.append(ASCII_CHARS.charAt(r.nextInt(ASCII_CHARS.length)))
-      }
-      b.toString
-    } else {
-      r.nextString(r.nextInt(options.maxStringLen))
+  def nextString(): String = {
+    val b = new StringBuilder(options.maxStringLen)
+    for (_ <- 0 until options.maxStringLen) {
+      b.append(options.validStringChars.charAt(r.nextInt(options.validStringChars.length)))
     }
+    b.toString
   }
 
-  private val ASCII_CHARS = "abcdefghijklmnopqrstuvwxyz"
+}
+
+object FuzzerOptions {
+  val ALPHABET_CHARS: String = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+  val NUMERIC_CHARS: String = "0123456789"
+  val ALPHANUMERIC_CHARS: String = ALPHABET_CHARS + NUMERIC_CHARS
+  val WHITESPACE_CHARS: String = " \t\r\n"
+  val SPECIAL_CHARS: String =
+    "!@#$%^&*()-+=/?,.<>\\|[]{}~;:`\"'"
+  val ALL_CHARS: String = ALPHABET_CHARS + NUMERIC_CHARS + SPECIAL_CHARS + WHITESPACE_CHARS
```

Comment:
Are we only considering ASCII here intentionally?  The name `ALL_CHARS` to me implies a wider coverage than it actually is.

Reply:
Also even if we're targeting ASCII, it's missing some ASCII control characters (e.g. characters for ordinal values 0x01, 0x02, etc.)

Reply:
I've removed these pre-defined sets of characters. The default now is to delegate to Scala's `Random.nextString` unless the fuzzer is created with a specific set of characters, as demonstrated in the CAST fuzzer tests in the PR.